chromium_src/chrome/browser/media/router/media_router_feature.cc

```diff
@@ -14,9 +14,11 @@
 namespace media_router {
 
 bool MediaRouterEnabled(content::BrowserContext* context) {
+#if !defined(OS_ANDROID) && !defined(OS_CHROMEOS)
```

Comment:
I think you can remove the `!defined(OS_CHROMEOS)` bit since Brave does not build there anyway, and also for consistency with what we had before (pre-Chromium 93), see https://github.com/brave/brave-core/blob/1.28.x/chromium_src/chrome/browser/media/router/media_router_feature.cc

Reply:
resolved